src/Entities/ArrowEntity.cpp

```diff
@@ -85,6 +85,7 @@ void cArrowEntity::OnHitSolidBlock(Vector3d a_HitPos, eBlockFace a_HitFace)
 
 	// Broadcast arrow hit sound
 	m_World->BroadcastSoundEffect("entity.arrow.hit", BlockHit, 0.5f, static_cast<float>(0.75 + (static_cast<float>((GetUniqueID() * 23) % 32)) / 64));
+	m_World->WakeUpSimulators(m_HitBlockPos);
```

Comment:
Why is this necessary? Add a comment maybe.

Reply:
Done.